crates/data_components/src/graphql/client.rs

```diff
@@ -49,9 +49,25 @@ enum DuplicateBehavior {
     Error,
 }
 
+type UnnestHandler = Box<dyn Fn(&Value) -> Result<Vec<Value>> + Send + Sync>;
+
+pub enum UnnestBehavior {
+    Default(usize),
+    Custom(UnnestHandler),
+}
+
+impl std::fmt::Debug for UnnestBehavior {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            UnnestBehavior::Default(depth) => write!(f, "Default({depth})"),
```

Comment:
```suggestion
            UnnestBehavior::Depth(depth) => write!(f, "Default({depth})"),
```

Just call this `Depth`